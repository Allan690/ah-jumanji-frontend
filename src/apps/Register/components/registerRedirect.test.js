import React from 'react';
import {shallow} from 'enzyme';

import RegisterRedirect from "./registerRedirect";

describe('Register redirect', () => {
 const mockRegisterRedirectFn = jest.fn;
 let wrapper;

 beforeEach(() => {
    wrapper = shallow(<RegisterRedirect login={mockRegisterRedirectFn} />);
  });

  it('should match the snapshoot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render register page correctly', () => {
    expect(wrapper.find('.row').exists()).toBe(true);
    expect(wrapper.find('p').text()).toBe('You were successfully registered.Check your mailbox for a link to activate your account.');
    expect(wrapper.find('p').length).toEqual(1);
  });
});
