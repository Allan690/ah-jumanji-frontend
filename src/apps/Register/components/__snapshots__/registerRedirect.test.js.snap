// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register redirect should match the snapshoot 1`] = `
<div
  className="row"
>
  <div
    className="bg-circle-1 bg-circle"
  />
  <div
    className="bg-circle-2 bg-circle"
  />
  <div
    className="bg-circle-3 bg-circle"
  />
  <div
    className="register-redirect-modal"
  >
    <img
      alt="success-icon"
      id="success-icon"
      src={null}
    />
    <p>
      You were successfully registered.
      <br />
      Check your mailbox for a link to activate your account.
    </p>
  </div>
</div>
`;
